mkdocs.yml

```
@@ -0,0 +1,130 @@
+# Extra
+extra:
+  social:
+    - icon: fontawesome/brands/github
+      link: https://github.com/JuanFKurucz/uruguay-news
+    - icon: fontawesome/brands/twitter
+      link: https://twitter.com/uruguaynews
+  analytics:
+    provider: google
+    property: G-XXXXXXXXXX
+  version:
+    provider: mike 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)